<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Kabloom_Mahesh" ignored="1" name="test1" tests="2" failures="1" timestamp="17 Jun 2019 09:36:42 GMT" time="0.035" errors="0">
  <testcase name="testPrintMessage" time="0.01" classname="com.testngscript.GroupTestExample"/>
  <testcase name="testingExitMessage" time="0.002" classname="com.testngscript.GroupTestExample">
    <failure type="java.lang.AssertionError" message="expected [www..com] but found [www.tutorialspoint.com]">
      <![CDATA[java.lang.AssertionError: expected [www..com] but found [www.tutorialspoint.com]
at com.testngscript.GroupTestExample.testingExitMessage(GroupTestExample.java:31)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testingExitMessage -->
  <testcase name="testSalutationMessage" time="0.0" classname="com.testngscript.GroupTestExample">
    <ignored/>
  </testcase> <!-- testSalutationMessage -->
</testsuite> <!-- test1 -->
